newm-server/src/main/kotlin/io/newm/server/database/migration/V54__CreateReleases.kt

```diff
@@ -0,0 +1,91 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+class V54__CreateReleases : BaseJavaMigration() {
```

Comment:
This should be `V58_UpdateReleases.kt`

It needs to have a single `ALTER TABLE...` SQL statement in it to add the new `pre_save_page` column to the releases table.